packages/server/src/utilities/rowProcessor/index.ts

```diff
@@ -216,10 +216,6 @@ export async function inputProcessing(
     if (field.type === FieldType.FORMULA) {
       delete clonedRow[key]
     }
-    // remove any AI values, they are to be generated
```

Comment:
Why this? Do we want to persist/analyse these imports?

Reply:
As I understood it, this was the thing stripping AI column results when doing a bulk import.

Reply:
that's it - when you re-import a CSV that has AI columns they were being stripped. We don't want to make the user generate them again, unless they specifically click to do so